src/CommunicationLayer/CommDeviceControl/ConnectionController.cpp

```diff
@@ -4,8 +4,7 @@ ConnectionController::ConnectionController(QString exchangeName
         ,  QString queueName
         ,  QString ipAddress
         ,  quint16 port)
-    : type_(CommDefines::Internet)
-    , exchangeName_(exchangeName)
+    : exchangeName_(exchangeName)
```

Comment:
Oh sorry! I meant that the `type_` should be checked before creating the `InternetConnectionService`, to leave potential for future communication protocols.